docker-compose.yml

```diff
@@ -0,0 +1,49 @@
+version: "3.4"
+
+services:
+  localstack-s3:
+    image: localstack/localstack:latest
+    environment:
+      - PORT_WEB_UI=8081
+      - DEFAULT_REGION=eu-west-2
+      - DATA_DIR=/tmp/localstack/data
+    ports:
+      - "8081:8081"
+
+  db:
+    image: postgres:11.2
+    environment:
+      POSTGRES_USER: postgres
+      POSTGRES_PASSWORD: example
+      POSTGRES_DB: airflow-db
+    healthcheck:
+      test: ["CMD-SHELL", "pg_isready -U postgres"]
+      interval: 30s
+      timeout: 30s
+      retries: 3
+
+  airflow_initdb:
+    &airflow
+    build:
+      target: dev
+      context: .
+    command: initdb
+    depends_on:
+      - db
+    environment:
+      WAIT_HOSTS: db:5432
+    healthcheck:
```

Comment:
I'd assume `initdb` exits once finished, so it doesn't need an healthcheck (the other `airflow_*` services do).

Reply:
Correct, it does exit. I added this so the setting is copied to the other services.

Reply:
Unused but it shouldn't necessarily hurt

Reply:
See suggestion on https://github.com/libero/elife-style-content-adapter-prototype/pull/1/files#r280103287 which would make this only available on `webserver`

Reply:
updated.